Presentation/PresentLogin.cs

```diff
@@ -1,63 +1,124 @@
                 {
-                    loginSuccesful = Login.TryLogin(email, password);
+                    Console.WriteLine("Enter your email:");
+                    string email = Console.ReadLine() ?? ""; // Get email (null check for potential empty input)
+
+                    validEmail = Login.IsValidEmail(email); // Validate email format
 
-                    if (loginSuccesful)
+                    if (validEmail) // If email is valid
                     {
-                        Console.WriteLine("Login successful!");
-                        
+                        Console.WriteLine("Enter your password:");
+                        string password = Console.ReadLine() ?? ""; // Get password (null check)
+
+                        validPassword = Login.IsValidPassword(password); // Validate password complexity
+
+                        if (validPassword) // If password is valid
+                        {
+                            try // Try block for potential exceptions during login
+                            {
+                                loginSuccessful = Login.TryLogin(email, password);
+
+                                if (loginSuccessful) // Login successful
+                                {
+                                    Console.WriteLine("Login successful!");
+                                    // PresentMovies.Start(); // Call to present movies (implementation not shown)
+                                    break; // Exit the loop
+                                }
+                                else // Login failed
+                                {
+                                    Console.WriteLine("Invalid email or password. Or your account does not exist.");
+                                }
+                            }
+                            catch (Exception ex) // Catch general exceptions for login errors
+                            {
+                                Console.WriteLine("Error during login: {0}", ex.Message);
+                            }
+                        }
+                        else // Password invalid
+                        {
+                            Console.WriteLine("Invalid password. Please check password complexity requirements.");
+                        }
                     }
-                    else
+                    else // Email invalid
                     {
-                        Console.WriteLine("Invalid email or password.");
+                        Console.WriteLine("Invalid email format. Please enter a valid email address.");
                     }
                 }
-                catch (Exception ex) // Catch a general Exception for errors
-                {
-                    Console.WriteLine("Error during login: {0}", ex.Message);
-                }
             }
-            else if (userChoice == "+")
+            else if (userChoice == "+") // Create superuser option selected
             {
```

Comment:
Is het niet beter om eerst een admin te zijn om dit te kunnen doen?

Reply:
ja idd kunnen later toevoegen dat er een condition word gechecked voordat een gebruiker een superuser kan aanmaken maar ik dacht voor nu kan het zo dan heb je in ieder geval wat